### ISSUE TITLE
Cannot read property 'Client' of undefined

### ISSUE THREAD

**Issue by helgetan** [bugsnag/bugsnag-js#604]
Hey, 
i currently have a Angular 7 Setup with ServerSide Rendering. 
In Browsermode evryting works fine and compiling to production also throws no errors, but
my Problem is when i start the server i get the following error: 

```
   __metadata("design:paramtypes", [_bugsnag_js__WEBPACK_IMPORTED_MODULE_1__["Bugsnag"].Client])
                                                                                         ^TypeError: Cannot read property 'Client' of undefined
    at Module../node_modules/@bugsnag/plugin-angular/dist/esm/index.js 
```

The Angular Setup is as following:

```
import bugsnag from '@bugsnag/js';
import {BugsnagErrorHandler} from '@bugsnag/plugin-angular';
const bugsnagClient = bugsnag(environment.APIKEYS_EXTERNAL.BUGSNAG);
export function errorHandlerFactory() {
  return new BugsnagErrorHandler(bugsnagClient)
}
@NgModule({
....
  providers: [ ...,{ provide: ErrorHandler, useFactory: errorHandlerFactory } ]
...
})
```

Maybe someone can help me
thank you :)

**Comment by mattdyoung** [comment]
Hi @helgetan 

We can't see anything wrong in your Angular setup there, so it's possible this is a bug in bugsnag-js.

Are you able to reproduce this with a minimal example project and share with us so we can reproduce and investigate further?

**Comment by VicenteT** [comment]
Hello @helgetan , did you fix? I have exactly the same problem but I don't know why. Until now It was working well and It works on my teammate computer's. I removed 'node_modules' folder and execute `npm install` again but nothing.

**Comment by mattdyoung** [comment]
Closing for now as unable to reproduce.

If someone has a standalone example project which reproduces the issue, please share with us and we'll investigate.

**Comment by VicenteT** [comment]
Hello @mattdyoung , I have exactly the same problem. And I don't know how. I can't reproduce this error on an example project because my project is big.

I have detected that in my server.js file this line fails:

![image](https://user-images.githubusercontent.com/5908623/66135485-53ade200-e5fa-11e9-9413-7be5654b27be.png)

I have been two weeks with this problem and I need to fix it urgently because it is slowing my job and I cant test or even create new builds to production. I wrote to customer service without answer.

For you knowledge:

- I have tried to remove and reinstall all dependencies.
- I have tried to remove my project and git clone again.
- It's an Angular 7 app with server rendering like the first comment.
- It started to happen two weeks ago.
- My teammate can generate builds without problem. He has Ubuntu, I don't know if this is the problem.
- I have a MacOS Mojave 10.14.6.

**Comment by abigail-bugsnag** [comment]
@VicenteT Thank you for the additional information. I have taken a look at this and have been able to reproduce the error that is being seen on this thread. 

This looks like it is a bug with the notifier, but have not yet identified the root cause. We are continuing to investigate and will update you when we know more.

**Comment by mattdyoung** [comment]
This is fixed in [v6.4.2](https://github.com/bugsnag/bugsnag-js/releases/tag/v6.4.2).
